docs/contributing/pull-requests/code-review.md

```diff
@@ -76,12 +103,14 @@ scenarios where discarding the reviews are generally seen as accepted.
 
 #### Approve
 
-Approving a PR means that you have confidence in that the code works, and that it does what the PR
+Approving a PR means that you have confidence in that the code works and that it does what the PR
 claims. This can be based on testing the change or on previous domain knowledge.
 
 - The PR targets the [correct branch](branching/#which-branching-model-to-choose).
 - You have verified that the linked Jira issue description matches the changes made in the PR.
 - You have read and understood the full impact of the changes suggested by the PR.
+- You have verified that all possible changes have been
+  [unit tested](./../testing/unit/naming-conventions.mdx).
```

Comment:
This is a major change to our current practices, which are less strict about this. This is worth highlighting when this is shared more broadly.